default_api.go

```diff
@@ -665,7 +665,7 @@ func (a *DefaultApiService) CreatePullRequest(projectKey, repositorySlug string,
 Creates a branch using the information provided in the {@link RestCreateBranchRequest request}  &lt;p&gt;  The authenticated user must have &lt;strong&gt;REPO_WRITE&lt;/strong&gt; permission for the context repository to call  this resource.
 
 @return */
-func (a *DefaultApiService) CreateBranch(projectKey, repositorySlug string, localVarPostBody interface{}) (*APIResponse, error) {
+func (a *DefaultApiService) CreateBranch(projectKey, repositorySlug string, PostBody interface{}) (*APIResponse, error) {
```

Comment:
instead of using the interface as it is, let's define the struct body. It will be helpful for the next user to know what are the body params needed by the create branch API.

Reply:
Addressed with commit 168662ab0c57c00b33fbbf3d28bb0ffe7e388f7b